packages/twenty-front/src/modules/information-banner/hooks/useAccountToReconnect.ts

```diff
@@ -0,0 +1,24 @@
+import { ConnectedAccount } from '@/accounts/types/ConnectedAccount';
+import { currentUserState } from '@/auth/states/currentUserState';
+import { InformationBannerKeys } from '@/information-banner/enums/InformationBannerKeys.enum';
+import { CoreObjectNameSingular } from '@/object-metadata/types/CoreObjectNameSingular';
+import { useFindOneRecord } from '@/object-record/hooks/useFindOneRecord';
+import { useRecoilValue } from 'recoil';
+
+export const useAccountToReconnect = (key: InformationBannerKeys) => {
+  const currentUser = useRecoilValue(currentUserState);
+
+  const userVars = currentUser?.userVars;
+
+  const firstAccountIdToReconnect = userVars?.[key]?.[0];
```

Comment:
Can we return early to avoid a query if not needed and firstAccountIdToReconnect is undefined/null? @bosiraphael